atlp-rwanda/atlp-pulse-fn
# Ft-create dashboard cards

### PR Description

- coordinator and manager cards should display all teams they manage, their average ratings and status of trainees

### Description of tasks that were expected to be completed

- Each team has its own card
- Cards are color-represented for example, red means the team is underperforming.
- All data points from the Figma design are implemented

### How has this been tested?

- checkout to branch ft-create-dashboard cards
- npm i
- npm run dev
- login as a coordinator or an admin or manager
- please make sure that you have  ttls so that you will be able to create a team 
- navigate to trainees on the sidebar and add some trainees
- navigate to ratings and rate different trainees
- navigate to teams
- you will be able to see teams cards with corresponding trainees and their ratings
